### ISSUE TITLE
Moved to in project plugin folder, game now takes a really long time in standalone and in package

### ISSUE THREAD

**Issue by USE-Nicholas** [truong-bui/AsyncLoadingScreen#108]
As the title says.
I removed the plugin from the marketplace folder and added the correct version from Git to the project folder. 
I did this as I could not package the game for Linux.
Now this did fix the problem, but the wait time for standalone and packaged game is has dramatically increased.

**Comment by truong-bui** [comment]
> ...
> Now this did fix the problem, but the wait time for standalone and packaged game is has dramatically increased.

Yeah, of course. When you move any plugins to the project Plugins folder, Unreal Engine recompiles them every time, which increases compiling time. The workaround is to take a cup of coffee and wait.

**Comment by USE-Nicholas** [comment]
I think you misunderstand.
The time it now takes for my project to load and to play in standalone has greatly increased, since moving the plugin and compiling.
